recipes/recastnavigation/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class RecastNavigationConan(ConanFile):
+    name = "recastnavigation"
+    homepage = "https://github.com/recastnavigation/recastnavigation"
+    description = " Navigation-mesh Toolset for Games"
+    topics = ("conan", "navmesh", "recast", "navigation", "crowd")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "ZLIB"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob('recastnavigation-*/')[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["RECASTNAVIGATION_DEMO"] = False
+        self._cmake.definitions["RECASTNAVIGATION_TESTS"] = False
+        self._cmake.definitions["RECASTNAVIGATION_EXAMPLES"] = False
+        self._cmake.definitions["RECASTNAVIGATION_STATIC"] = not self.options.shared
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+
+    def package_info(self):
+        self.cpp_info.components["Recast"].name = "Recast"
```

Comment:
I think you want to only set the name of the `cmake_find_package` and `cmake_find_package_multi`.
```suggestion
        self.cpp_info.components["Recast"].names["cmake_find_package"] = "Recast"
        self.cpp_info.components["Recast"].names["cmake_find_package_multi"] = "Recast"
```
Same goes for the other components.